app/template_statistics/rest.py

```diff
@@ -27,31 +29,17 @@
 
 @template_statistics.route('')
 def get_template_statistics_for_service_by_day(service_id):
```

Comment:
As there is new logic here should there be some more tests added?

Reply:
there have been more tests added. lots of them. all of https://github.com/alphagov/notifications-api/blob/a166a16c096b5570c4fe546b05a3153adbd6ad97/tests/app/template_statistics/test_rest.py#L30-L246 are tests for this function